src/Nethermind/Nethermind.Db/SimpleFilePublicKeyDb.cs

```diff
@@ -60,20 +61,20 @@ public SimpleFilePublicKeyDb(string name, string dbDirectoryPath, ILogManager lo
                 }
                 else
                 {
-                    _cache.AddOrUpdate(key, newValue => Add(value), (x, oldValue) => Update(oldValue, value));
+                    _cache.AddOrUpdate(key.ToArray(), newValue => Add(value), (x, oldValue) => Update(oldValue, value));
```

Comment:
Why `.ToArray` here?

Reply:
Can't use span as a template for the function. So can't migrate this method.